cluster/images/etcd/Makefile

```diff
@@ -114,5 +116,124 @@ ifeq ($(ARCH),amd64)
 	gcloud docker -- push $(REGISTRY)/etcd:$(REGISTRY_TAG)
 endif
 
-all: build
-.PHONY:	build push
+ETCD2_ROLLBACK_NEW_TAG=3.0.17
+ETCD2_ROLLBACK_OLD_TAG=2.2.1
+
+# Test a rollback to etcd2 from the earliest etcd3 version.
+test-rollback-etcd2:
+	mkdir -p $(TEMP_DIR)/rollback-etcd2
+	cd $(TEMP_DIR)/rollback-etcd2
+
+	@echo "Starting $(ETCD2_ROLLBACK_NEW_TAG) etcd and writing some sample data."
+	docker run -ti -v $(TEMP_DIR)/rollback-etcd2:/var/etcd \
```

Comment:
Can we spell out the docker flags?  e.g. `--tty`, `--interactive`?  Maybe I just haven't run docker in too long.

Reply:
Updated.